src/style.css

```diff
@@ -0,0 +1,157 @@
+body {
+  margin: 0;
+}
+
+.container {
+  position: relative;
+  left: 0.5%;
+  right: 0.5%;
+  top: 0.25%;
+  bottom: 0.25%;
+  width: 198px;
+  height: 406px;
+  box-sizing: border-box;
+}
+
+.container--card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-left: 16px;
+  padding-right: 16px;
+}
+
+.container__picture {
+  position: absolute;
+  width: 160px;
+  height: 134px;
+  left: calc(50% - 160px / 2);
+  top: 32px;
+}
+
+.container__link-1 {
+  position: absolute;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  text-decoration: none;
+  margin-top: 206px;
+  margin-right: 16px;
+}
+
+.container__label-1 {
+  position: absolute;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-top: 246px;
+}
+
+.starss {
+  flex-direction: row;
+  display: flex;
+  background: #fff;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  margin-top: 276px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  display: flex;
+  padding-left: 4px;
+}
+
+.stars--4 > :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.starss__label {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  display: flex;
+  align-items: flex-end;
+  color: #060b35;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 72px;
+  margin-top: 24px;
+}
+
+.price__1 {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  align-items: flex-end;
+  color: #616070;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.price__2 {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  align-items: flex-end;
+  text-align: right;
+  color: #060b35;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.container__link-2 {
+  position: absolute;
+  padding: 12px 55px;
+  bottom: 16px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+  text-decoration: none;
+  background: #00acdc;
+  border-radius: 5px;
+  margin-top: 350px;
+  width: 54px;
+}
+
+.container__link-2:hover {
+  width: 54px;
+
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  background: #fff;
+  color: #00acdc;
```

Comment:
Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover